### PR TITLE
Add ability to change a hackathon's logo/banner

### PR DESCRIPTION
**Pull request by cjdenio**
https://github.com/hackclub/hackathons-backend/assets/34525547/d38839ff-9d43-459c-b624-308cecc1e58e

### CODE REVIEW

File: app/views/admin/hackathons/show.html.erb [hunk 1]
```diff
@@ -14,19 +14,31 @@
...
 
     <% if @hackathon.logo.present? || @hackathon.banner.present? %>
-      <section style="display: flex;" class="pt3">
+      <section style="display: flex; gap: var(--spacing-2)" class="pt3">
         <% if @hackathon.logo.present? %>
-          <div>
-            <h2 class="heading--small">Logo</h2>
-            <%= image_tag @hackathon.logo, size: 128, style: "background-color: black;" %>
-          </div>
+          <turbo-frame id="logo">
```
Comment (northeastprince) [inline app/views/admin/hackathons/show.html.erb:19]:
```suggestion
          <turbo-frame id="hackathon_logo">
```

File: app/views/admin/hackathons/show.html.erb [hunk 2]
```diff
@@ -14,19 +14,31 @@
...
+            </div>
+          </turbo-frame>
         <% end %>
 
         <% if @hackathon.banner.present? %>
-          <div>
-            <h2 class="heading--small">Banner</h2>
-            <%= image_tag @hackathon.banner, height: 128, style: "background-color: black;" %>
-          </div>
+          <turbo-frame id="banner">
```
Comment (northeastprince) [inline app/views/admin/hackathons/show.html.erb:32]:
```suggestion
          <turbo-frame id="hackathon_banner">
```

**Review by northeastprince — COMMENTED** [review]
Thanks for this! I was thinking it could be cool if we could make it so you can click on the image itself to overwrite it (darken and add an icon on hover though for clarity). Not sure how the styling would work, but I've seen it done before so it's definitely possible.

File: app/views/admin/hackathons/show.html.erb (hunk 1 above)
Comment (cjdenio) [reply app/views/admin/hackathons/show.html.erb:19]: This is intentional because the generated ID for the `file_field` is `hackathon_logo`

**Review by flexcommits — APPROVED** [review]
LGTM

**Pull request merged**